pkg/analytics/models.go

```diff
@@ -14,7 +14,8 @@ import (
 type EventType string
 
 const (
-	JobEventType = "job_v1"
+	JobEventType       = "job_v1"
+	SubmitJobEventType = "submit_job_v1"
```

Comment:
based on feedback from Peter, lets name these `bacalhau.job_v1.start` and `bacalhau.job_v2.stop` or something on those lines